src/client/components/forms/userCollection.js

```diff
@@ -0,0 +1,306 @@
+		}
+		else {
+			submissionURL = '/collection/create/handler';
+		}
+		request.post(submissionURL)
+			.send(data)
+			.promise()
+			.then((res) => {
+				if (res.status === 200) {
+					window.location.href = `/collection/${res.body.id}`;
+				}
+				else {
+					this.setState(prevState => ({
+						collection: prevState.collection,
+						errorText: `Internal Error: ${res.body.error}`
+					}));
+				}
+			});
+	}
+
+	isValid() {
+		return this.name.getValue() && this.entityType.getValue();
+	}
+
+	getCleanedCollaborators() {
+		const cleanedCollaborators = this.state.collection.collaborators.filter(collaborator => collaborator && collaborator.id !== null);
+		return cleanedCollaborators;
+	}
+
+	handleAddCollaborator() {
+		const newCollaborators = this.state.collection.collaborators;
+		newCollaborators.push({
+			id: null,
+			name: ''
+		});
+		this.setState(prevState => ({
+			collection: {
+				...prevState.collection,
+				collaborators: newCollaborators
+			},
+			errorText: prevState.errorText
+		}));
+	}
+
+	handleRemoveCollaborator(index) {
+		const newCollaborators = this.state.collection.collaborators;
+		newCollaborators.splice(index, 1);
+		this.setState(prevState => ({
+			collection: {
+				...prevState.collection,
+				collaborators: newCollaborators
+			},
+			errorText: prevState.errorText
+		}));
+	}
+
+	handleChangeCollaborator(newCollab, index) {
+		let newCollaborator;
+		if (!newCollab) {
+			newCollaborator = {
+				id: null,
+				name: ''
+			};
+		}
+		else {
+			newCollaborator = newCollab;
+		}
+		const newCollaborators = this.state.collection.collaborators;
+		newCollaborators[index] = newCollaborator;
+		this.setState(prevState => ({
+			collection: {
+				...prevState.collection,
+				collaborators: newCollaborators
+			},
+			errorText: prevState.errorText
+		}));
+	}
+
+	render() {
+		if (this.state.collection.collaborators.length === 0) {
+			this.handleAddCollaborator();
+		}
+
+		const privacyOptions = ['Private', 'Public'].map((option) => ({
+			name: option
+		}));
+		const entityTypeOptions = ['Author', 'Work', 'Edition', 'Edition-Group', 'Publisher'].map((entity) => ({
+			name: entity
+		}));
+		const initialName = this.state.collection.name;
+		const initialDescription = this.state.collection.description;
+		const initialPrivacy = this.state.collection.public ? 'Public' : 'Private';
+		const initialType = this.state.collection.entityType;
+		const {errorText} = this.state;
+		const errorAlertClass =
+			classNames('text-center', 'margin-top-1', {hidden: !errorText});
+		const submitLabel = this.props.collection.name ? 'UPDATE' : 'CREATE';
+
+		/* eslint-disable react/jsx-no-bind */
+		return (
+			<Grid>
+				<h1>Create a collection</h1>
+				<Row>
+					<Col md={12}>
+						<p className="lead">Create Your Collection</p>
+					</Col>
+				</Row>
+				<Row>
+					<Col
+						id="collectionForm"
+						md={8}
+						mdOffset={2}
+					>
+						<form
+							className="form-horizontal"
```

Comment:
I would replace this with `padding-sides-0`, which I prevented some differences in padding between form elements and buttons/h elements outside of from-groups.